BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -512,12 +512,19 @@ open class BraintreeClient @VisibleForTesting internal constructor(
     }
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    fun getSubstringAfterKey(jsonString: String, key: String): String {
-        val keyWithQuotes = "\"$key\":\""
-        val startIndex = jsonString.indexOf(keyWithQuotes) + keyWithQuotes.length
-        val endIndex = jsonString.indexOf("\",", startIndex)
-        return if (startIndex != -1 && endIndex != -1) {
-            jsonString.substring(startIndex, endIndex)
+    fun getGraphQLMutationName(jsonString: String): String {
```

Comment:
We have ticket item for this in iOS, or is that something different?

Reply:
Yep, we have a ticket `DTMOBILES`-683, last sprint, last sprint, I checked this out a bit, but I think there's some kind of redirection that's stopping us from getting the query. I'll need to dig into it more